packages/vats/src/lib-board.js

```diff
@@ -375,3 +380,29 @@ export const makeBoard = (initSequence = 0, options = {}) => {
   const make = prepareBoardKit(makeScalarBigMapStore('baggage'));
   return make(initSequence, options).board;
 };
+
+/**
+ * @param {import('@agoric/zone').Zone} zone
+ */
+export const makeRecorderFactory = zone => {
```

Comment:
we use `prepare` to convey that a durable kind is being defined, signaling that is must be called in the first crank of restart

```suggestion
export const prepareRecorderFactory = zone => {
```

Reply:
thanks. was suffering pretty bad naming fatigue at the time.